pkg/integration/tests/custom_commands/form_prompts.go

```diff
@@ -37,7 +37,7 @@ var FormPrompts = NewIntegrationTest(NewIntegrationTestArgs{
 							{
 								Name:        "bar",
 								Description: "Bar",
-								Value:       "BAR",
+								Value:       `"BAR"`,
```

Comment:
Why are we quoting this? Shouldn't we not have quotes in the value so that we can ensure that it's properly quoted by the quote function?

Reply:
This is tested to be escaped `"` itself. Without `| Quote`, only `BAR` is output even if `value` is <code>&#096;"BAR"&#096;</code>.

Reply:
Ah I see: we need the quotes in the value so that when call the quote function the quotes are escaped and end up in the diff. Okay that makes sense!